x-pack/plugins/security_solution/server/lib/overview/query.dsl.ts

```diff
@@ -142,57 +142,167 @@ export const buildOverviewHostQuery = ({
+                          },
+                          {
+                            term: {
+                              'event.category': 'driver',
+                            },
+                          },
+                        ],
+                      },
+                    },
+                    {
+                      term: {
+                        'endgame.event_type_full': 'image_load_event',
+                      },
+                    },
+                  ],
                 },
               },
             },
             network_event_count: {
               filter: {
-                term: {
-                  'endgame.event_type_full': 'network_event',
+                bool: {
+                  should: [
+                    {
+                      bool: {
+                        filter: [
+                          {
+                            bool: {
+                              must_not: {
+                                term: { 'network.protocol': 'dns' },
+                              },
+                            },
+                          },
+                          {
+                            term: { 'event.category': 'network' },
+                          },
+                        ],
+                      },
+                    },
+                    {
+                      term: {
+                        'endgame.event_type_full': 'network_event',
+                      },
+                    },
+                  ],
                 },
               },
             },
             process_event_count: {
               filter: {
-                term: {
-                  'endgame.event_type_full': 'process_event',
+                bool: {
+                  should: [
+                    {
+                      term: { 'event.category': 'process' },
+                    },
+                    {
+                      term: {
+                        'endgame.event_type_full': 'process_event',
+                      },
+                    },
+                  ],
                 },
               },
             },
             registry_event: {
               filter: {
-                term: {
-                  'endgame.event_type_full': 'registry_event',
+                bool: {
+                  should: [
+                    {
+                      term: { 'event.category': 'registry' },
+                    },
+                    {
+                      term: {
+                        'endgame.event_type_full': 'registry_event',
+                      },
+                    },
+                  ],
                 },
               },
             },
             security_event_count: {
               filter: {
-                term: {
-                  'endgame.event_type_full': 'security_event',
+                bool: {
+                  should: [
+                    {
+                      bool: {
+                        filter: [
```

Comment:
Security events for endpoint should be an array of two values for `event.category == [session, authentication]`